### ISSUE TITLE
Library ESP8266WiFi.h missing in WebSocketClient_Demo.ino

### ISSUE THREAD

**Issue by silvestret** [morrissinger/ESP8266-Websocket#16]
I have an Arduino UNO R3 and a module ESP8266 connected(RX= pin 3 and  TX= pin 2).
I attached an image of my sketch.
Does your library will work with this sketch? if so, where can i get library ESP8266WiFi.h ?

Thanks
![esquema a arduino 1](https://cloud.githubusercontent.com/assets/2334611/16550684/f30a7b28-417b-11e6-86bd-79148327dedd.jpg)

**Comment by morrissinger** [comment]
Are you trying to load this sketch onto the Arduino then? It doesn't have the right architecture for this sketch. This needs to be loaded onto the esp directly.

**Comment by silvestret** [comment]
Oh ok, so the board in arduino would be esp8266. I got the esp8266 of this photo.
![s-l225](https://cloud.githubusercontent.com/assets/2334611/16565572/3af67db4-41dc-11e6-9a68-0d23499cd428.jpg)

So, your library is for esp8266  board, and it will not work with my esp8266 arduino module.
I am right?

**Comment by morrissinger** [comment]
Exactly. You have to set the board in your IDE to ESP8266, which is going to draw on a different set of libraries. Then, the way I have done it is to hook up the ESP through an FTDI board/cable to the USB on my laptop. If I recall, there is a pin on the ESP you have to pull down in order to get the chip to accept a firmware flash.

**Comment by silvestret** [comment]
Thank you so much for your response!

Greetings

**Comment by karanmakharia** [comment]
@morrissinger What if I want to load the sketch into Arduino? Do you know any other library that will work with this. Because the GPIO pins on ESP are not enough, so I will have to use Arduino board probable UNO or MEGA
